A52-xds-custom-lb-policies.md

```diff
@@ -312,5 +312,9 @@ The `weighted_target` configuration will be:
 }
 ```
 
+### Temporary environment variable protection
+During initial development, this feature will be enabled by the `GRPC_EXPERIMENTAL_ENABLE_CUSTOM_LB_CONFIG` environment variable. If unset or not true, the `load_balancing_policy` field in xDS Cluster proto will not be considered and load balancing policy configuration will be based on the `lb_policy` field. This environment variable protection will be removed once the feature has proven stable.
```

Comment:
`GRPC_EXPERIMENTAL_XDS_CUSTOM_LB_CONFIG`?

Reply:
I did already implement it with the value I specified, but I can change if you feel it's worth it.

Reply:
It seems we have no standard here.

- `GRPC_XDS_EXPERIMENTAL_ENABLE_RING_HASH `
- `GRPC_XDS_EXPERIMENTAL_RBAC`
- `GRPC_EXPERIMENTAL_XDS_RLS_LB`
- `GRPC_EXPERIMENTAL_ENABLE_OUTLIER_DETECTION `

So I guess whatever is fine; good thing they're temporary. 🤷

Reply:
This should be `GRPC_EXPERIMENTAL_XDS_CUSTOM_LB_CONFIG`.

For the first several of these, we put `XDS` before `EXPERIMENTAL`, but the usage of this kind of env var is not really xDS-specific, so I think it makes sense to use `GRPC_EXPERIMENTAL_` as our default prefix for all such env vars.  And in this case, the feature is specifically xDS-based custom LB configs, so `XDS` should be in the name.

Reply:
Updated to use GRPC_EXPERIMENTAL_XDS_CUSTOM_LB_CONFIG.